azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java

```diff
@@ -255,35 +254,19 @@ public static void main(final String[] args) throws ExecutorManagerException {
     // TODO : Revisit this logic with full implementation for JMXBEanManager and other callback mechanisms
     JmxJobMBeanManager.getInstance().initialize(azkabanProps);
     // execute the flow
-    flowContainer.submitFlow(execId, currentDir, projectZipName);
+    flowContainer.submitFlow(execId, currentDir);
   }
 
-  private void setupWorkDir(final Path currentDir, final String projectZipName)
-          throws ExecutorManagerException {
-    // Move files to respective dirs
+  private void setupWorkDir(final Path currentDir) throws ExecutorManagerException {
```

Comment:
Can you please rename this method to setupExecutionDir? Is this still needed? What is flowPreparer.setupContainerizedExecution doing?

Reply:
Yes I can rename it. FlowPreparer is downloading the zip file.

Reply:
What is this method is doing? This is creating executions directory right?

Reply:
Yes it is creating the execution directory but not the execution(s) directory and FlowPreparer is putting the zip file in it.